blog/2021-05-17-WritingFastConstraintStreamsSecretRecipe.adoc

```diff
@@ -0,0 +1,193 @@
+= Writing fast constraints with OptaPlanner: The secret recipe
+:page-interpolate: true
+:awestruct-author: triceo
+:awestruct-layout: blogPostBase
+:awestruct-tags: [constraint, insight, performance]
+
+Do you want OptaPlanner to run faster?
+Do you want to increase your score calculation speed, reaching great solutions sooner?
+Let me show you how to optimize your
+https://docs.optaplanner.org/latest/optaplanner-docs/html_single/index.html#constraintStreams[Constraint Streams]
+constraints for performance and scalability.
+Turns out you only need to remember one advice:
+
+== DO LESS
```

Comment:
Personal opinion: I wouldn't write all caps. "Do less" works just as fine. All caps is shouting.